packages/cli/test/fixtures/discover/index.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+exports.SANDBOX_FILES = [
+  {
+    path: '',
+    file: 'test-schema-defs.json',
+    content: fs.readFileSync(
+      require.resolve('./models/test-schema-defs.json.txt'),
```

Comment:
I am confused, why are you using `.json.txt` extension? Can we store JSON content in files with `.json` extension please?

Reply:
Done